osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/PlayerArea.cs

```diff
@@ -101,9 +102,12 @@ public void LoadScore(Score score)
             stack.Push(new MultiSpectatorPlayerLoader(Score, () =>
             {
                 var player = new MultiSpectatorPlayer(Score, SpectatorPlayerClock);
-                player.OnGameplayStarted += () => OnGameplayStarted?.Invoke();
 
-                ClockAdjustmentsFromMods.BindAdjustments(player.ClockAdjustmentsFromMods);
+                player.OnGameplayStarted += () =>
+                {
+                    clockAdjustmentsFromMods.BindAdjustments(player.ClockAdjustmentsFromMods);
```

Comment:
Make sure you test this to make sure nothing funny happens with moving it in here.

Reply:
I did not mean to make this change.